packages/jetpack-redirects/README.md

```diff
@@ -0,0 +1,54 @@
+# Jetpack Redirects
+
+A helper function to build URLs using the `jetpack.com/redirect` service.
+
+## Usage
+
+`getRedirectUrl( required source, optional args );`
+
+### $source (required)
+
+Source can be either a “source handler” or a URL.
+
+A “source handler” must be registered in the Jetpack Redirects service, on the server side. It’s a slug that points to an URL that may or may not have dynamic parts in it.
+
+A “URL” is a string that must start with “https://" and doesn’t need to be registered on the server. However, if it is registered, it will point to the URL set as target there rather than to the source. (Note: It will only work for whitelisted domains)
+
+### $args (optional)
+
+This is optional and allows you to pass an array (or object in JS) with more parameters to build the URL:
+
+* **site**: This is used to identify the site and also to fill in the `[site]` placeholder in the target. 
+
+* **path**: Optional. Used to fill in the `[path]` placeholder in the target.
+
+* **query**: Optional. A string with additional variables to be added in the query string. Must be passed as a string in `key=value&foo=bar` format.
+
+* **anchor**: Optional. An anchor to be added to the final URL. Must be a single string. Example: `section1`
+
+## Examples
+
+### Example 1
+
+`getRedirectUrl( 'jetpack', { query: 'foo=bar', anchor: 'section' } );`
+
+This will create: `https://jetpack.com/redirect?source=jetpack&anchor=section&query=foo%3Dbar`
+
+That will point to: `https://jetpack.com/?foo=bar#section`
```

Comment:
I'd describe more clearly that:
- X is an URL that `getRedirectUrl` returns
- Y is an URL that a redirect URL will redirect to when the redirect URL is accessed

Instead of "point to", let's say "redirect to". The server will send a HTTP 302 redirect response.

Reply:
improved in https://github.com/Automattic/wp-calypso/pull/41497/commits/32db0e43145b36e35be4c12952ec7abe9029941d